app/app.vue

```diff
@@ -46,67 +49,129 @@ onPrehydrate(() => {
     }
   }
 })
+
+const darkMode = useCookie('dark-mode', {
```

Comment:
I love it!

we should probably move to local storage plus onPrehydrate as the page is prerendered

(or use nuxt colour mode module)

Reply:
Maybe we don't really need to store it at all :cat2: